cluster-provision/okd/scripts/provision.sh

```diff
@@ -208,15 +208,24 @@ oc adm policy add-cluster-role-to-user cluster-admin admin
 
 if [ "${CNAO}" == "true" ]; then
     # Apply network addons
-    oc create -f /manifests/cna/namespace.yaml
-    oc create -f /manifests/cna/network-addons-config.crd.yaml
-    oc create -f /manifests/cna/operator.yaml
-    oc create -f /manifests/cna/network-addons-config-example.cr.yaml
-
-     # Wait until all the network components are ready
-    until oc wait networkaddonsconfig cluster --for condition=Available --timeout=100s; do
-        sleep 10
+
+    # TODO: pass CNAO version at gocli
+    cnao_version=0.25.0
+    cnao_path=/manifests/cnao/$cnao_version/
+    oc create -f $cnao_path/namespace.yaml
+    oc create -f $cnao_path/network-addons-config.crd.yaml
+    oc create -f $cnao_path/operator.yaml
+
+    # Wait until cnao is ready
+    oc wait deployment -n cluster-network-addons cluster-network-addons-operator --for condition=Available --timeout=5m
```

Comment:
possible that when we move to version 0.27 5 minutes wont be enough from what i saw locally